python/src/SobolIndicesAlgorithmImplementation_doc.i.in

```diff
@@ -56,10 +52,10 @@ Let us define
 
 .. math::
 
-   \begin{array}{ccc}
-    VT_i & = & \sum_{I \ni i} V_I \\
-    V_{-i} & = & \Var{ \Expect{Y \vert X_1, \ldots, X_{i-1}, X_{i+1}, \ldots X_{n_X}} }.
-   \end{array}
+    VT_i   & = \sum_{I \ni i} V_I \\
+    V_{-i} & = \Var{ \Expect{Y \vert X_1, \ldots, X_{i-1}, X_{i+1}, \ldots X_{n_X}} }.
+
+for :math:`i=1,\ldots,n_X`.
```

Comment:
Why not 

"Let us define for :math:`i=1,\ldots,n_X`

[formulas]"

Notice there is a point at the end of line 56 so it is supposed to be the end of the sentence.

Reply:
Ok. This needs to be fixed.

Reply:
Done.